#! /bin/sh
patch -p0 -f $* < $0
exit $?

2009-04-08 [colin]      3.7.1cvs41

        * src/gtk/manage_window.c
                Disable transient windows on win32 - seems to
                fix most focusing issues

Index: src/gtk/manage_window.c
===================================================================
RCS file: /srv/cvs/claws-mail/claws/src/gtk/manage_window.c,v
retrieving revision 1.1.4.8
retrieving revision 1.1.4.9
diff -u -p -u -r1.1.4.8 -r1.1.4.9
--- src/gtk/manage_window.c	9 Jan 2009 17:47:17 -0000	1.1.4.8
+++ src/gtk/manage_window.c	8 Apr 2009 16:03:42 -0000	1.1.4.9
@@ -67,9 +67,8 @@ void manage_window_destroy(GtkWidget *wi
 
 void manage_window_set_transient(GtkWindow *window)
 {
-	/* debug_print("manage_window_set_transient(): window = %p, focus_window = %p\n",
-		    window, focus_window); */
-
+#ifndef G_OS_WIN32
 	if (window && focus_window)
 		gtk_window_set_transient_for(window, GTK_WINDOW(focus_window));
+#endif
 }
